src/labels/CustomLabels.labels

```diff
@@ -2752,6 +2752,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>- Household Duplicate</value>
     </labels>
     <labels>
+        <fullName>hhErrorInvalidContactFieldName</fullName>
+        <categories>Households, Household Naming</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>hhErrorInvalidContactFieldName</shortDescription>
+        <value>The field, {0}, does not match any field name on Contact.</value>
```

Comment:
@smarovich @RadSFDeveloper We should be clear that it's the API Name here, it's so confusing for folks between the Label, Developer Name and API and that's where I often see them trip up here.